osCam/userconfig/forms.py

```diff
@@ -105,69 +38,51 @@ class CameraViewForm( ModelForm ):
         required=False,
         label='mirror image',
         widget=forms.CheckboxInput())
-    codec = forms.ChoiceField(
-        choices=(('h264', 'h264'), ('mjpeg', 'mjpeg'), ('avi', 'avi')),)
-
+    scale = forms.ChoiceField( 
+        choices=((1.00, '1.00'), (0.75, '0.75'), (0.50, '0.50'), (0.25, '0.25')), )
     class Meta:
         model = CameraView
         fields = (
             'showMotionBoxes',
-            'showContours',
             'showText',
             'text',
-            'contrast',
-            'brightness',
-            'recording',
             'fps',
             'invert',
             'mirror',
-            'codec',)
```

Comment:
looks like codec is being removed? assuming it's because there won't be enough time to implement different codecs this semester. I remember you guys discussed it at last scrum/meeting but I forgot all the details

Reply:
I removed a bunch of settings that are currently not being used and won't be used in the near future as they their use is not implementation is not high priority and they were just adding complexity and taking up space. if we want to put them back in later we can but for now everything is much cleaner without them.

As our MVP is based on the raspberry pi running Ubuntu the default codec and file type works (and as far as I know should work for all other major OS distributions, I believe we have tested on Mac, but I don't know if anyone has tested on windows.